src/Components/Assets/AssetManage.tsx

```diff
@@ -75,25 +69,34 @@ const AssetManage = (props: AssetManageProps) => {
   const fetchData = useCallback(
     async (status: statusType) => {
       setIsLoading(true);
-      const assetData = await dispatch(getAsset(assetId));
+      // const assetData = await dispatch(getAsset(assetId));
+      const { data: assetData } = await request(routes.getAsset, {
+        pathParams: {
+          external_id: assetId,
+        },
+      });
       if (!status.aborted) {
         setIsLoading(false);
-        if (assetData?.data) {
-          setAsset(assetData.data);
+        if (assetData) {
+          setAsset(assetData);
 
           const transactionFilter = assetData.qr_code_id
             ? { qr_code_id: assetData.qr_code_id }
             : { external_id: assetId };
 
           const [transactionsData, servicesData] = await Promise.all([
```

Comment:
Does this really work? `request` returns an object that contains `res`, `data` and all instead of `res` alone.